src/crepe/api/LoopTimer.cpp

```diff
@@ -0,0 +1,63 @@
+
+#include "../facade/SDLContext.h"
+#include "../util/log.h"
+
+#include "LoopTimer.h"
+
+using namespace crepe;
+
+LoopTimer::LoopTimer() {
+	dbg_trace();
+}
+
+LoopTimer & LoopTimer::get_instance() {
+    static LoopTimer instance;
+    return instance;
+}
+
+void LoopTimer::start() {
+    last_frame_time = SDLContext::get_instance().get_ticks();
+    elapsed_time = 0;
```

Comment:
Hier worden members gereferenced zonder `this->` (zie code standaard), dit komt veel vaker voor dus check even alles hierop